tutorials/fundamentals/trim0.md

```diff
@@ -0,0 +1,42 @@
+To remove leading zeroes from a number string in JavaScript, you can simply multiply the string by 1.
+
+```javascript
+let x = '0042';
+x = x * 1;
+x; // 42
+```
+
+## parseInt() and parseFloat()
+
+Another way to remove leading zeroes is to use the `parseInt()` method, provided the string is not a `float`.
+Use `parseFloat()` if the number string is a float.
+
+```javascript
+let x = '0042';
+x = parseInt(x);
```

Comment:
Make sure to do `parseInt(x, 10)`. Otherwise you end up with whacky behavior with stuff like `parseInt('0x42')`.